frontend/src/core/editor/components/FailedChecks.js

```diff
@@ -125,11 +125,11 @@ export class FailedChecksBase extends React.Component<InternalProps> {
                         className='close'
                         onClick={this.closeFailedChecks}
                     >
-                        ×
+                        X
```

Comment:
No need to transform this. While it is a button, it has its own className of `.close` which does not transform the text.